tests/forbid_new_submodules_test.py

```diff
@@ -31,12 +33,32 @@ def git_dir_with_git_dir(tmpdir):
 )
 def test_main_new_submodule(git_dir_with_git_dir, capsys, cmd):
     subprocess.check_call(cmd)
-    assert main() == 1
+    assert main(()) == 0
+    assert main(('random_non-related_file',)) == 0
+    assert main(('foo',)) == 1
+    out, _ = capsys.readouterr()
+    assert out.startswith('foo: new submodule introduced\n')
+
+
+def test_main_new_submodule_committed(git_dir_with_git_dir, capsys):
+    REV_PARSE_HEAD = ('git', 'rev-parse', 'HEAD')
+    FROM = subprocess.check_output(REV_PARSE_HEAD).decode().strip()
```

Comment:
why are you capitalizing local variables

Reply:
Fixed `FROM` and `TO`, preserved `REV_PARSE_HEAD` because it's a constant. If you insist, I can put it to the top of the module (since it only needed in this function I've put it there).